modules/admission-webhook-types.adoc

```diff
@@ -0,0 +1,124 @@
+// Module included in the following assemblies:
+//
+// * architecture/admission-controllers.adoc
+
+[id="admission-webhook-types_{context}"]
+= Types of admission webhooks
+
+Cluster administrators can include _mutating admission webhooks_ or _validating admission webhooks_
+in the admission chain of the API server.
+
+[[mutating-admission-webhooks]]
+*Mutating admission webhooks* are invoked during the mutation phase of the admission process, which allows modification of the resource content before it is persisted. One example of a mutating admission webhook is the
+ifdef::openshift-enterprise,openshift-origin[]
+Pod Node Selector
+endif::openshift-enterprise,openshift-origin[]
+ifdef::openshift-online,openshift-dedicated[]
+Pod Node Selector
+endif::openshift-online,openshift-dedicated[]
+feature, which uses an annotation on a namespace to find a label selector and add it to the pod specification.
+
+[[mutating-admission-webhooks-sample]]
+.Sample mutating admission webhook configuration:
+
+[source,yaml]
+----
+apiVersion: admissionregistration.k8s.io/v1beta1
```

Comment:
The YAML highlighting is weird in this and the below code block. I think that it's improperly indented (that apiVersion should be at the same level as kind)

Reply:
Well spotted, the preceding hyphen was missing. I have corrected this.